Sources/Preprocessors/DefaultAnimationPreprocessor.swift

```diff
@@ -249,9 +259,9 @@ class DefaultAnimationPreprocessor: BasePreprocessor {
       if animators.contains(where: { $0.canAnimate(view: toView, appearing: true) || $0.canAnimate(view: fromView, appearing: false) }) {
         defaultAnimation = .none
       } else if inNavigationController {
-        defaultAnimation = presenting ? .push(direction:.left) : .pull(direction:.right)
+        defaultAnimation = presenting ? .push(direction:.leading) : .pull(direction:.trailing)
```

Comment:
@ManueGE The only thing that concerns me is now we're changing the behavior of this animation. I agree that it's nice to change things based on the `userInterfaceLayoutDirection` -- but a Hero user could be relying on this not being changed

Reply:
You are totally right, I understand that some users could be relying on this even if maybe it's not the "right" behavior. I saw it as a bug fix, but I understand your point.

Reply:
Can we make this configurable so that users can opt-in to the old behavior?

Reply:
Let me think about it a little bit. Will try to figure out something tomorrow

Reply:
I kept the "left to right" default behavior, but "user interface" and "right to left" behaviors can be set using the `defaultTransitionDirectionStrategy` property of `HeroTransition`.

Reply:
Thanks @ManueGE. I skimmed this earlier this week. This looks totally reasonable 👍 Will review tomorrow and hopefully merge + release!